recipe/build.sh

```diff
@@ -30,10 +30,21 @@ fi
     --arch="${PYTHIA_ARCH}" \
     --prefix="${PREFIX}" \
     --with-lhapdf6="${PREFIX}" \
+    --with-fastjet3="${PREFIX}" \
     --with-gzip="${PREFIX}" \
     --with-mg5mes
 
 make install --jobs="${CPU_COUNT}"
+make clean
+
+# Remove documentation and examples from share/Pythia8 as not needed for runtime.
+# Keep share/Pythia8/examples/Makefile.inc as that might be looked for.
+# Keep share/Pythia8/pdfdoc as required at runtime.
+mv $PREFIX/share/Pythia8/examples/Makefile.inc .
+rm -rf $PREFIX/share/Pythia8/examples/*
+mv Makefile.inc $PREFIX/share/Pythia8/examples/Makefile.inc
+rm -rf $PREFIX/share/Pythia8/htmldoc
+rm -rf $PREFIX/share/Pythia8/pdfdoc
```

Comment:
These changes halve the size of the distribution. It is currently just over 40 MB and this brings it down to

```
----------------------------------------------
linux-64/pythia8-8.312-py312hf2d3f55_301.conda
----------------------------------------------
-- Size: 20.7MB

```